src/providers/Gardens.js

```diff
@@ -78,10 +78,18 @@ function mergeGardenMetadata(garden) {
     ...garden.token,
     ...metadata.token,
   }
+  const wrappableToken = garden.wrappableToken
```

Comment:
let's use the one line syntax `garden.wrappableToken  && {  ...garden.wrappableToken,        ...metadata.wrappableToken  }`